client/scss/components/_promo.scss

```diff
@@ -183,12 +209,56 @@
   color: color('accessible-grey-1');
   min-width: 0;
   flex: 1;
+  position: relative;
+
+  .promo--standalone & {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    padding: 1rem;
+    margin-top: -3 * $vertical-space-unit;
+    margin-bottom: $vertical-space-unit;
+    background: color('white');
+    border-radius: $border-radius-unit;
+
+    @include respond-to('medium') {
+      margin-top: -6 * $vertical-space-unit;
+    }
+  }
 
   @include container-query(('l': ('8', '12'), 'm': ('8'))) {
     padding-right: map-get($gutter-width, 'large');
   }
 }
 
+.promo__heading,
```

Comment:
Why have we done it this way?
Does:
```sass
.promo--standalone {
  .promo__heading, .promo__copy {}
}
```

Make more sense as it is to do with the standalone? I wonder if this could be applied to a lot of the classes above and below?